public/index.html

```diff
@@ -324,6 +324,51 @@ <h1>Rudr: A Kubernetes Implementation of the Open Application Model</h1>
         <p><a href="https://github.com/oam-dev/rudr" class="button is-primary is-large"><span class="icon"><i class="fa fa-github"></i></span> &nbsp; Learn More</a></p>
       </div>
     </div>
+
+    <div class="columns">
+      <div class="column is-quarter is-one-quarter-mobile has-text-left"><img src="images/alibaba-cloud.png" alt="Rudr" /></div>
+      
+      <div class="column is-three-quarters is-three-quarters-mobile">
+        <h1>Alibaba</h1>
+        <small><a href="https://www.alibabacloud.com/">www.alibabacloud.com</a></small>
+      </div>
+      
+    </div>
+
+    <div class="columns">
+      <div class="column is-quarter has-text-left">
+      </div>
+      <div class="column is-three-quarters">
+
+        <p>Alibaba supports Open Application Model in their offerings:</p>
+        <ul>
+          <li><b>Alibaba Enterprise Distributed Application Service</b> (Available) - A PaaS platform for a variety of application deployment options and microservices solutions.</li>

```

Comment:
can you help add a link here  https://www.alibabacloud.com/product/edas

Reply:
Done, see suggestion below